lib/puppet/pops/validation/checker4_0.rb

```diff
@@ -649,13 +684,13 @@ def check_ResourceBody(o)
   end
 
   def check_ResourceDefaultsExpression(o)
-    if o.form && o.form != :regular
+    if o.form != 'regular'
```

Comment:
Yes. The default value of the field is 'regular' and its type stipulates an Enum (not optional).